components/raftstore/src/store/peer.rs

```diff
@@ -6204,7 +6271,54 @@ pub fn make_transfer_leader_response() -> RaftCmdResponse {
 
 // The Raft message context for a MsgTransferLeader if it is a reply of a
 // TransferLeader command.
-pub const TRANSFER_LEADER_COMMAND_REPLY_CTX: &[u8] = &[1];
+const TRANSFER_LEADER_COMMAND_REPLY_CTX: &[u8] = &[1];
```

Comment:
How about replacing `TAG_COMMAND_REPLY` with `TRANSFER_LEADER_COMMAND_REPLY_TAG`.
```suggestion
const TRANSFER_LEADER_COMMAND_REPLY_TAG: u8 = 1;
const TRANSFER_LEADER_COMMAND_REPLY_CTX: &[u8] = &[TRANSFER_LEADER_COMMAND_REPLY_TAG];
```

Reply:
The constant `TRANSFER_LEADER_COMMAND_REPLY_CTX` is kept on purpose, it is used to test the compatibility.

---

Changing to `TRANSFER_LEADER_COMMAND_REPLY_TAG` might be verbose, for example, using it requires writing `TransferLeaderContext::TRANSFER_LEADER_COMMAND_REPLY_TAG` where the term "transfer leader" is repeated.

Reply:
Get.